thunder/tests/coverage_tests/test_coverage_trace.py

```diff
@@ -59,6 +59,7 @@ def get_dummy_input(model_name, config):
         return {"input_ids": torch.randint(0, 1000, (1, 16), device="cpu")}
 
 
+@pytest.mark.skip(reason="https://github.com/Lightning-AI/lightning-thunder/issues/2436")
```

Comment:
> Since this PR is only for the diffuser test, can we review and merge this one first, and then open a separate PR to fix test_coverage_trace.py (https://github.com/Lightning-AI/lightning-thunder/issues/2436) ?

Quite surely I'm missing something, becuase it's not likely that the proposition here is "break and disable all coverage test for transformer in order to enable coverage test for diffuser"?

Reply:
> It's not like disable the `test_coverage_trace`

of they are all skipped it the same as disabled :)